src/meta/model/src/actor_dispatcher.rs

```diff
@@ -57,79 +55,3 @@ impl From<DispatcherType> for PbDispatcherType {
         }
     }
 }
-
```

Comment:
Can we create a migration to drop the table and also remove this file?

Reply:
Shall we keep the `dispatcher` table for a release? The backfill of `fragment_relation` depends on the `dispatcher` table. I'm afraid that if something wrong happened after we drop the `dispatcher` table and the `fragment_relation` table gets rollbacked and also dropped in the `down`, then we will lose the dispatcher information forever and cannot backfill the `fragment_relation` table.

Reply:
>  I'm afraid that if something wrong happened after we drop the `dispatcher` table and the `fragment_relation` table gets rollbacked and also dropped in the `down`

🤔 The backfill of `fragment_relation` is done in previous migration `m20250106_072104_fragment_relation.rs`. Actually the logic of "down" is mostly untrustworthy and irreversible, we will not execute it or rely on it to perform a downgrade. Currently, the only consistency that can be guaranteed in rw is `backup` and `restore`. The only issue I can think of is that `fragment_relation` is not included in the backup, which has been fixed in this PR.

It's fine to keep it for now, as it doesn't have any negative effects and aids in recovering abnormal data and pinpointing issues.

Reply:
Sure. I will create an issue to track it.